package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "group-income",
   "version": "0.0.1",
-  "contractsVersion": "0.1.6",
+  "contractsVersion": "0.1.9",
```

Comment:
Please set this to `0.2.0` per the reasons described here: https://github.com/okTurtles/group-income/pull/1722#discussion_r1323344309